diff --git a/py/py.mk b/py/py.mk
index bac38f074..2e6f3d9c6 100644
--- a/py/py.mk
+++ b/py/py.mk
@@ -216,7 +216,7 @@ PY_CORE_O = $(addprefix $(BUILD)/, $(PY_CORE_O_BASENAME))
 PY_EXTMOD_O = $(addprefix $(BUILD)/, $(PY_EXTMOD_O_BASENAME))
 
 # this is a convenience variable for ports that want core, extmod and frozen code
-PY_O = $(PY_CORE_O) $(PY_EXTMOD_O)
+PY_O += $(PY_CORE_O) $(PY_EXTMOD_O)
 
 # object file for frozen code specified via a manifest
 ifneq ($(FROZEN_MANIFEST),)
